qpc/report/conftest.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public License,
+# version 3 (GPLv3). There is NO WARRANTY for this software, express or
+# implied, including the implied warranties of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv3
+# along with this software; if not, see
+# https://www.gnu.org/licenses/gpl-3.0.txt.
+#
```

Comment:
we don't add LICENSE headers anymore :sunglasses:

Reply:
cc @infinitewarp we still need to do some cleanup on this repo. Did you use any script to help cleaning up quipucords?

Reply:
Unfortunately, no. I just did a simple search for `Copyright` across the project in VS Code, and I aggressively clicked through files and manually deleted all the headers. 😂 Sure, I _could_ have automated it, but whatever time I spent finding just the right with pattern match in `sed` or `perl` or whatever probably would have taken just a long, and I didn't anticipate repeating that operation ever again.

Reply:
I removed the LICENSE header only on the new file created. I have an upcoming pr that fixes the log messages for the whole project, I will include a cleanup for the rest of LICENSE headers on that one.

Reply:
did you push the changes? the header is still here :sweat_smile: